pkg/controllers/localbuild/argo.go

```diff
@@ -5,105 +5,47 @@ import (
 	"embed"
 
 	"github.com/cnoe-io/idpbuilder/api/v1alpha1"
-	"github.com/cnoe-io/idpbuilder/pkg/k8s"
-	"github.com/cnoe-io/idpbuilder/pkg/util"
-	appsv1 "k8s.io/api/apps/v1"
-	corev1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/runtime/schema"
 	ctrl "sigs.k8s.io/controller-runtime"
-	"sigs.k8s.io/controller-runtime/pkg/client"
-	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
-	"sigs.k8s.io/controller-runtime/pkg/log"
 )
 
 //go:embed resources/argo/*
-var installFS embed.FS
+var installArgoFS embed.FS
 
 const (
-	argoApplicationControllerName string = "argocd-application-controller"
-	argoServerName                string = "argocd-server"
-	argoRepoServerName            string = "argocd-repo-server"
+	argocdNamespace string = "argocd"
 )
 
-func GetRawInstallResources() ([][]byte, error) {
-	return util.ConvertFSToBytes(installFS, "resources/argo")
-}
-
-func GetK8sInstallResources(scheme *runtime.Scheme) ([]client.Object, error) {
-	rawResources, err := GetRawInstallResources()
-	if err != nil {
-		return nil, err
-	}
-
-	return k8s.ConvertRawResourcesToObjects(scheme, rawResources)
-}
-
-func newArgoNamespace() *corev1.Namespace {
-	return &corev1.Namespace{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: "argocd",
+func (r LocalbuildReconciler) ReconcileArgo(ctx context.Context, req ctrl.Request, resource *v1alpha1.Localbuild) (ctrl.Result, error) {
```

Comment:
Let's use pointer receiver. I know we don't need pointers for these but rest of them do. We are currently mixing value and pointer receivers.

Reply:
good call. not sure how it fell through